mtp_noms_ops/templates/mtp_common/user_admin/update.html

```diff
@@ -1,3 +1,69 @@
-{% extends 'mtp_common/user_admin/update.html' %}
+{% extends 'mtp_common/user_admin/base.html' %}
+{% load i18n %}
+{% load mtp_common %}
 
-{% block body_classes %}{{ block.super }} mtp-page-with-staff-email-input{% endblock %}
+{% block page_title %}{{ page_title }} – {{ block.super }}{% endblock %}
+
+{% block admin_content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <header>
+        <h1 class="govuk-heading-xl">{{ page_title }}</h1>
+      </header>
+
+      {% if form.create %}
+        <p class="govuk-body-lead">
+          {% trans 'Fill in a user’s details to create an account.' %}
+          {% trans 'You can give them a general user or an account management role.' %}
+        </p>
+        {% if permissions_note %}
+          <p>{{ permissions_note }}</p>
+        {% endif %}
+      {% endif %}
+
+      <form action="." method="post" novalidate>
+        {% csrf_token %}
+
+        {% include 'govuk-frontend/components/error-summary.html' with form=form only %}
+
+        {% if form.create %}
+          {% include 'mtp_common/forms/field.html' with field=form.username input_classes='govuk-input--width-20' only %}
+        {% else %}
+          <p>
+            <span class="govuk-label" id="id_username-label">{% trans 'Username' %}</span>
+            <strong aria-labelledby="id_username-label">{{ form.username.value }}</strong>
+          </p>
+        {% endif %}
+
+        {% include 'mtp_common/forms/field.html' with field=form.first_name input_classes='govuk-input--width-20' only %}
+        {% include 'mtp_common/forms/field.html' with field=form.last_name input_classes='govuk-input--width-20' only %}
+        {% include 'mtp_common/forms/field.html' with field=form.email input_classes='govuk-input--width-20' only %}
+
+        {% with field=form.role %}
+          {% if field.field.choices %}
```

Comment:
Not sure if you need this `if` statement here. In fact you can probably let the form widget render the field itself:

Reply:
I didn't try to be clever, I copied from `mtp-common` and override only what needed to change to be on the safe side.